app/services/user_grouper.rb

```diff
@@ -12,8 +13,20 @@ def call
     [me_item, internal_user_item, client_user_item, inactive_user_item].compact
   end
 
+  def user_filter(label, column, scope = nil)
+    default_options = { input_label: label, column: column, include_blank: false, blank_value_label: 'All Users' }
+    default_options[:scope] = scope if scope
+    return searchable_options(default_options) if max_dropdown_size_exceeded?(base_users)
```

Comment:
I'm manually changing the filter type here, right now I don't believe the search scope stuff works with grouped dropdowns, so in this case I would prefer the grouped dropdown if possible, but fallback to searchable dropdown if max exceeded.

I think we should be able to figure out how to support this in future, but just wanted to get us one step closer for now